### ISSUE TITLE
Sublimetext

### ISSUE THREAD

**Issue by BClare19** [Installomator/Installomator#623]
Sublimetext no longer downloading and installing

Seems they don't link directly to the ZIP anymore. They use a thanks page which then directs the browser to the specific platform's URL. After discussing on slack

https://www.sublimetext.com/download_thanks?target=mac

Thanks

**Comment by pythoninthegrass** [comment]
Working through local testing with `assemble.sh`. Will have a PR by EOD.

**Comment by pythoninthegrass** [comment]
Didn't have time to fully debug the error output while downloading:
```
2022-07-14 19:34:19 : REQ   : sublimetext : ################## Start Installomator v. 9.2, date 2022-05-16
2022-07-14 19:34:19 : INFO  : sublimetext : ################## Version: 9.2
2022-07-14 19:34:19 : INFO  : sublimetext : ################## Date: 2022-05-16
2022-07-14 19:34:19 : INFO  : sublimetext : ################## sublimetext
2022-07-14 19:34:19 : INFO  : sublimetext : BLOCKING_PROCESS_ACTION=prompt_user
2022-07-14 19:34:19 : INFO  : sublimetext : NOTIFY=success
2022-07-14 19:34:19 : INFO  : sublimetext : LOGGING=INFO
2022-07-14 19:34:20 : INFO  : sublimetext : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2022-07-14 19:34:20 : INFO  : sublimetext : Label type: zip
2022-07-14 19:34:20 : INFO  : sublimetext : archiveName: Sublime Text.zip
2022-07-14 19:34:20 : INFO  : sublimetext : no blocking processes defined, using Sublime Text as default
2022-07-14 19:34:20 : INFO  : sublimetext : name: Sublime Text, appName: Sublime Text.app
2022-07-14 19:34:20 : WARN  : sublimetext : No previous app found
2022-07-14 19:34:20 : WARN  : sublimetext : could not find Sublime Text.app
2022-07-14 19:34:20 : INFO  : sublimetext : appversion:
2022-07-14 19:34:20 : INFO  : sublimetext : Latest version of Sublime Text is Build 4126
2022-07-14 19:34:20 : REQ   : sublimetext : Downloading  to Sublime Text.zip
2022-07-14 19:34:20 : ERROR : sublimetext : error downloading
2022-07-14 19:34:20 : ERROR : sublimetext : File list:
2022-07-14 19:34:20 : ERROR : sublimetext : File type: Sublime Text.zip: cannot open `Sublime Text.zip' (No such file or directory)
2022-07-14 19:34:20 : INFO  : sublimetext : App not closed, so no reopen.
2022-07-14 19:34:20 : ERROR : sublimetext : ERROR: Error downloading  error:
* Closing connection -1
curl: (3) URL using bad/illegal format or missing URL

2022-07-14 19:34:20 : REQ   : sublimetext : ################## End Installomator, exit code 2
```
Fragment:
```
sublimetext)
    # credit: Søren Theilgaard (@theilgaard); Lance Stephens (@pythoninthegrass)
    name="Sublime Text"
    type="zip"
    downloadURL=$(curl -fs https://www.sublimetext.com/download_thanks?target=mac | grep -io -m 1 "https://download.*_mac.zip" | head -1)
    appNewVersion=$(curl -fs https://www.sublimetext.com/download | grep -i -A 4 "id.*changelog" | grep -io "Build [0-9]*")
    expectedTeamID="Z6D26JE4Y4"
    ;;
```
Suspect it breaks [downstream](https://github.com/scriptingosx/Installomator/blob/main/Installomator.sh#L5410). Will circle back tomorrow.

**Comment by scriptingosx** [comment]
closed with #593
